docs/medical-api/api-reference/patient/get-patient-consolidated.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: "Get Patient's Consolidated Data"
+description: "Gets a Patient's Consolidated Data."
```

Comment:
nit: should we consider making this `Get Patient's Consolidated Data in a FHIR Bundle`?

So that it's immediately obvious what this does